KomaMRICore/Project.toml

```diff
@@ -17,7 +17,6 @@ ThreadsX = "ac1d9e8a-700a-412c-b207-f0111f4b6c0d"
 Adapt = "3, 4"
 CUDA = "3, 4, 5"
 Functors = "0.4"
-KomaMRIBase = "0.8"
```

Comment:
Revert this.

Reply:
Resolved in [7f1cc52](https://github.com/JuliaHealth/KomaMRI.jl/pull/184/commits/7f1cc52925d3be8803c4ff0c95c1e06f391b7c83)